api-reference/_hidden/GridBaseColumn/setCellValue.md

```diff
@@ -19,7 +19,7 @@ A read-only parameter providing access to the current row data.
 Return a promise for an asynchronous operation or return nothing.
 
```

Comment:
Line 7 (above):
`Specifies a function to be invoked after the user has edited a cell value, but before it will be saved in the data source.`

Can we update this sentence as follows?

`Specifies a function to be invoked after the user has edited a cell value, but before it is saved in the data source.`

Reply:
Okay